demo.html

```diff
@@ -11,11 +11,11 @@
   <link rel="stylesheet" type="text/css" href="assets/css/locuszoom.css"/>
 
   <script type="text/javascript">
-    LocusZoom.DefaultDataSources = {
-        position: new LocusZoom.Data.AssociationSource("http://portaldev.sph.umich.edu/api_internal_dev/v1/single/"),
-        ld: new LocusZoom.Data.LDSource("http://portaldev.sph.umich.edu/api_internal_dev/v1/pair/LD/"),
-        gene: new LocusZoom.Data.GeneSource("http://portaldev.sph.umich.edu/api_internal_dev/v1/annotation/genes/")
-    };
+    var apiBase = "http://portaldev.sph.umich.edu/api_internal_dev/v1/";
+    var ds = new LocusZoom.DataSources();
+	ds.addSource("base", ["AssociationLZ", {url:apiBase + "single/", params: {analysis: 3}}]);
```

Comment:
So unlike "LD" and "Gene" the positions/base/associations datasource/datalayer/panel seem to have a lot of names. Well, not that many. And naming things is one of the [two hard things](http://martinfowler.com/bliki/TwoHardThings.html). But does it feel like there's any convergence on a common name here? Quite probably all the names in play are totally legit for how they're used. Suffice it to say when I named that one panel the `PositionsPanel` I wasn't sure if it was the best name, and I'm still not. So if there is a more globally-applicable name for the main stuff rising above the rest I'd be happy to help apply it more globally.